dev/tools/tokens/material-tokens.json

```diff
@@ -0,0 +1,708 @@
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display-medium.tracking": {
+    "value": 0.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display-medium.size": {
+    "value": 45.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display-medium.weight": "weightRegular",
+  "md.sys.typescale.display-medium.font": "brandRegular",
+  "md.sys.typescale.display-large": "displayLarge",
+  "md.sys.typescale.display-large.line-height": {
+    "value": 64.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display-large.tracking": {
+    "value": 0.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display-large.size": {
+    "value": 57.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display-large.weight": "weightRegular",
+  "md.sys.typescale.display-large.font": "brandRegular",
+  "md.sys.typescale.subtitle2": "subtitle2",
+  "md.sys.typescale.subtitle1": "subtitle1",
+  "md.sys.typescale.headline6": "headline6",
+  "md.sys.typescale.display1": "display1",
+  "md.sys.typescale.display1.tracking": {
+    "value": 0.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle2.line-height": {
+    "value": 20.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle2.tracking": {
+    "value": 0.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle2.size": {
+    "value": 14.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle1.line-height": {
+    "value": 24.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle1.tracking": {
+    "value": 0.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle1.size": {
+    "value": 16.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.headline6.line-height": {
+    "value": 24.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.headline6.tracking": {
+    "value": 0.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.headline6.size": {
+    "value": 18.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display1.line-height": {
+    "value": 76.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.display1.size": {
+    "value": 64.0,
+    "units": "POINTS"
+  },
+  "md.sys.typescale.subtitle2.weight": "weightMedium",
+  "md.sys.typescale.subtitle2.font": "plainMedium",
+  "md.sys.typescale.subtitle1.weight": "weightMedium",
+  "md.sys.typescale.subtitle1.font": "plainMedium",
+  "md.sys.typescale.headline6.weight": "weightRegular",
+  "md.sys.typescale.headline6.font": "brandRegular",
+  "md.sys.typescale.display1.weight": "weightRegular",
+  "md.sys.typescale.display1.font": "brandRegular",
+  "md.ref.typeface.brand-display-regular": "Roboto Regular",
+  "md.ref.typeface.weight-bold": 700.0,
+  "md.ref.typeface.weight-medium": 500.0,
+  "md.ref.typeface.weight-regular": 400.0,
+  "md.ref.typeface.plain-medium": "Roboto Medium",
+  "md.ref.typeface.plain-regular": "Roboto Regular",
+  "md.ref.typeface.brand-medium": "Roboto Medium",
+  "md.ref.typeface.brand-regular": "Roboto Regular",
+  "md.comp.fab.primary.pressed.icon.color": "onPrimaryContainer",
```

Comment:
Is the idea that this json file would become longer each time we add a new component? Would it make sense to split this into multiple JSON files per component?

Reply:
My current thinking it to keep it simple with just a single file. That way each template has access to all the system parameters (elevations, opacity, etc) in a single map. If this gets unwieldy we can of course break them out into a json file per component (and one that has all the common system level stuff in it that can be used by all of them).